docs/index.html

```diff
@@ -0,0 +1,103 @@
+          "@floating-ui/utils/dom": "https://ga.jspm.io/npm:@floating-ui/utils@0.2.2/dist/floating-ui.utils.dom.mjs",
+          "@lit/reactive-element": "https://ga.jspm.io/npm:@lit/reactive-element@2.0.4/development/reactive-element.js",
+          "@lit/reactive-element/decorators/": "https://ga.jspm.io/npm:@lit/reactive-element@2.0.4/development/decorators/",
+          "@patternfly/pfe-core/controllers/": "https://ga.jspm.io/npm:@patternfly/pfe-core@3.0.0/controllers/",
+          "lit": "https://ga.jspm.io/npm:lit@3.1.3/index.js",
+          "lit-element/lit-element.js": "https://ga.jspm.io/npm:lit-element@4.0.5/development/lit-element.js",
+          "lit-html": "https://ga.jspm.io/npm:lit-html@3.1.3/development/lit-html.js",
+          "lit-html/": "https://ga.jspm.io/npm:lit-html@3.1.3/development/",
+          "lit/": "https://ga.jspm.io/npm:lit@3.1.3/",
+          "tslib": "https://ga.jspm.io/npm:tslib@2.6.2/tslib.es6.mjs"
+        }
+      }
+    }
+    </script>
+    <script type="module">
+      import '@patternfly/elements/pf-icon/pf-icon.js';
+      import '@rhds/elements/rh-tooltip/rh-tooltip.js';
+      import '@rhds/elements/rh-surface/rh-surface.js';
+      import '@rhds/elements/rh-button/rh-button.js';
+      import '@rhds/elements/rh-alert/rh-alert.js';
+    </script>
+
+    <script type="module">
+      import Toast from './javascript/toast.js';
+      
+      async function copy(html) {
+        await navigator.clipboard.writeText(html)
+      }
+
+      const tooltips = document.querySelectorAll('rh-tooltip');
+      tooltips.forEach(tip => {
+        const tipButton = tip.querySelector('rh-button');
+        tipButton.addEventListener('click', async (event) => {
+          const { html, icon } = event.target.dataset;
+          console.log(event.target.dataset);
+          copy(html); 
+          await Toast(`Copied ${icon}`);
+        });
+      })
+    </script>    
+  </head>
+  <body>
+    <header><h1>Red Hat Icons</h1></header>
+    <main>
+      {% for set in iconSets %}
+      {% set id = set.set | lower %}
+
+      {% if set.set === "ui" %}
+        {% set section = set.set | upper %} 
+      {% else %}
+        {% set section = set.set | capitalize %}
+      {% endif %}
+
+      <section class="{{ set.set }}" aria-labelledby="{{ id }}">
+        <h2 id="{{ id }}">{{ section }}</h2>
+        {% for icon in set.icons %}
+        <figure>
+          {% include set.dir + '/' + icon + '.svg' %}
+          <figcaption>
+            {{ icon }}
+             <rh-tooltip position="bottom">
+              <rh-button name="toast" variant="link" data-html='<rh-icon set="{{ set.set }}" icon="{{ icon }}"></rh-icon>' data-icon="{{icon}}">
+                <pf-icon set="far" icon="copy" size="md" loading="eager"></pf-icon>
+                <span class="sr-only">Click to</span> Copy <span class="sr-only">rh-icon {{icon}} from set {{set.set}}
```

Comment:
This is [**very** verbose](https://github.com/user-attachments/assets/224741a7-4584-43a4-b8c9-9506a9b1e087). Right now, it's:

> Click to copy rh-icon arrow-down from set microns

...repeated 100's of times.

Consider changing it to:

"Copy arrow-down"

or:

"Copy arrow down markup"

("markup" could be "code", "icon", etc)